spec/core/ics-026-routing-module/UPGRADES.md

```diff
@@ -86,7 +83,8 @@ function onChanUpgradeTry(
   channelIdentifier: Identifier,
   counterpartyPortIdentifier: Identifier,
   counterpartyChannelIdentifier: Identifier,
-  counterpartyVersion: string) => (version: string, err: Error) {
+  counterpartyVersion: string,
+  previousVersion: string) => (version: string, err: Error) {
```

Comment:
Should this be `previousCounterpartyVersion`?

Reply:
No, `counterpartyVersion` is the new version the counterparty is proposing to upgrade to

`previousVersion` is the current version of our chain before the upgrade. Maybe we should rename to current version?

Reply:
Ah okay, thanks, sounds good to me!

Reply:
I think `previousVersion` makes sense for naming, but maybe we could have `previousVersion` come before `counterpartyVersion` in the list of args? Would it makes more sense for the "self" to be stated before the counterparty. It doesn't make too much of a difference to me either way